fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -224,6 +224,17 @@ private void bootstrapDerbyDatabases(PropertyGroup fhirConfig) throws Exception
                 log.info("No '" + datasourceJndiName + "' dataSource found; skipping OAuth client table bootstrapping");
             }
 
+            try {
+                // Check the batch database, if the batch database configuration is there, and available.
+                ds = (DataSource) ctxt.lookup("jdbc/fhirbatchDB");
```

Comment:
what if we already configured either DB2 or postgresql as job repo? will this still try to bootstrap derby job repo ...

Reply:
and the same question for the OAUTH bootstraping...

Reply:
only if you have enabled boostrap as true.

Reply:
but our default setting is true, maybe better to check if the configured datasource is derby before bootstrap it ...

Reply:
I follow you. I am just testing my locallized fix (which is in boostrapBatchDb(ds)